docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -42,6 +42,8 @@ Malware protection levels are:
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
+Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure specific applications on the blocklist, refer to <<blocklist>>.
```

Comment:
Combining elements from both reviews:
```suggestion
Malware protection also allows you to implement a blocklist to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
```